themes/prism-shades-of-purple.css

```diff
@@ -0,0 +1,190 @@
+/**
+ * Shades of Purple Theme for Prism.js
+ *
+ * @author Ahmad Awais <https://github.com/AhmadAwais>
+ * @support Follow/tweet at https://twitter.com/MrAhmadAwais/
+ */
+
+code[class*='language-'],
+pre[class*='language-'] {
+	color: #9efeff;
+	direction: ltr;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
+
+	-moz-tab-size: 4;
+	-o-tab-size: 4;
+	tab-size: 4;
+
+	-webkit-hyphens: none;
+	-moz-hyphens: none;
+	-ms-hyphens: none;
+	hyphens: none;
+
+	font-family: 'Operator Mono', 'Fira Code', Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace !important;
+	font-weight: 400 !important;
+	font-size: 17px !important;
+	line-height: 25px !important;
+	letter-spacing: 0.5px !important;
```

Comment:
Why are the `!important`s needed here?

Reply:
Many many code themes for like WordPress try to put `!important` tags on code tokens by default. This is why I had to override them with this.